cax/tasks/data_mover.py

```diff
@@ -30,18 +31,79 @@ def copy(self, datum_original, datum_destination, method, option_type):
             server = config_original['hostname']
             username = config_original['username']
 
+        nstreams = 16
+
         # Determine method for remote site
         if method == 'scp':
```

Comment:
Here, we'd check `if 'scp' in methods`.
We should check GRID first, otherwise scp.